.github/workflows/deploy.yml

```diff
@@ -0,0 +1,89 @@
+name: Build and deploy
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build_md:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: install elan
+        run: |
+          curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- -y --default-toolchain $(cat lean-toolchain)
+          echo "$HOME/.elan/bin" >> $GITHUB_PATH
+
+      - name: build markdown files by mdgen
+        run: lake exe mdgen lean md
```

Comment:
```suggestion
        run: lake run build
```

Reply:
`lake run build` raises an error in GitHub Action (error message said that external command was not allowed..). so this should be as it is

Reply:
Interesting! Does that not happen locally when running it? I'd check myself but I have some issues building projects at the moment it seems (which are related to my setup not mdgen)

Reply:
There were no errors when I ran `lake run build` locally.